src/DependencyInjection/BitBagSyliusWishlistExtension.php

```diff
@@ -28,6 +28,7 @@ public function load(array $config, ContainerBuilder $container): void
         $this->registerResources('bitbag_sylius_wishlist_plugin', 'doctrine/orm', $config['resources'], $container);
         $loader->load('services.yml');
         $container->setParameter('bitbag_sylius_wishlist_plugin.parameters.wishlist_cookie_token', $config['wishlist_cookie_token']);
+        $container->setParameter('bitbag_sylius_wishlist_plugin.parameters.wishlist_expiration_period', $config['wishlist_expiration_period']);
```

Comment:
You could try to create a datetime object from this interval, so you validate the interval is correct. Otherwise, you fail early at container build time. IMHO it's a better developer experience to have the error at build then when running the clean command